src/Domain/Block.cpp

```diff
@@ -40,10 +41,10 @@ void Block<VolumeDim, TargetFrame>::pup(PUP::er& p) {
 
 template <size_t VolumeDim, typename TargetFrame>
 std::ostream& operator<<(std::ostream& os,
-                         const Block<VolumeDim, TargetFrame>& block) {
+                         const Block<VolumeDim, TargetFrame>& block) noexcept {
   os << "Block " << block.id() << ":\n";
```

Comment:
Should all `"\n"`s be changed to `'\n'`s? because this one isn't - is it because of the `:`?

Reply:
The single quotes is only for single characters (`\n` is a single character, a newline), but `:\n` is two characters.